ldm/dream/image_util.py

```diff
@@ -35,7 +35,7 @@ def resize(self,width=None,height=None) -> Image:
         # rw and rh are the resizing width and height for the image
         # they maintain the aspect ratio, but may not completelyl fill up
         # the requested destination size
-        (rw,rh) = (width,int(width/ar)) if im.width>=im.height else (int(height*ar),width)
+        (rw,rh) = (width,int(width/ar)) if im.width>=im.height else (int(height*ar),height)
```

Comment:
This was an unfortunate bug that caused images to be scaled down more aggressively than needed.